client/settings/fraud-protection/components/protection-levels/index.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+import HelpOutlineIcon from 'gridicons/dist/help-outline';
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { useCurrentProtectionLevel, useCurrencies } from 'wcpay/data';
+import {
+	FraudProtectionHelpText,
+	HighFraudProtectionModal,
+	StandardFraudProtectionModal,
+} from '../index';
+
+const ProtectionLevels = () => {
+	const [ isStandardModalOpen, setStandardModalOpen ] = useState( false );
+	const [ isHighModalOpen, setHighModalOpen ] = useState( false );
+	const { currencies } = useCurrencies();
+	const storeCurrency = currencies.default ? currencies.default : {};
+
+	const [
+		currentProtectionLevel,
+		updateProtectionLevel,
+	] = useCurrentProtectionLevel();
+
+	const handleLevelChange = ( level ) => {
+		updateProtectionLevel( level );
+	};
+
+	return (
+		<fieldset>
+			<ul>
+				<li>
+					<label htmlFor="fraud-protection__standard-level">
```

Comment:
Found an issue writing the advanced section, this label's `htmlFor` attribute needs to match the child input's `id` value to be able to get selected via clicking the text. It's not currently selectable via clicking the label.

Reply:
I need to work on this some more. I just noticed that now the modals don't open correctly any more as a result of this change. They require double clicking and the rendering looks off. I'll see what I can do.

Reply:
Okay, fixed for real in [2d6aa3c](https://github.com/Automattic/woocommerce-payments/pull/5450/commits/2d6aa3cdc05974a29d189a513404bb22e8b0f4bd).